src/test/kotlin/com/terraformation/backend/documentproducer/db/VariableImporterTest.kt

```diff
@@ -0,0 +1,820 @@
+      val expectedSelectRow =
+          VariableSelectsRow(
+              variableId = tableColumnVariableRow1.id!!,
+              variableTypeId = VariableType.Select,
+              isMultiple = false)
+
+      val actualSelectOptionRows =
+          variableSelectOptionsDao.fetchByVariableId(tableColumnVariableRow1.id!!)
+      val expectedSelectOptionRows =
+          listOf(
+              VariableSelectOptionsRow(
+                  variableId = tableColumnVariableRow1.id!!,
+                  variableTypeId = VariableType.Select,
+                  name = "Validation/verification",
+                  position = 1),
+              VariableSelectOptionsRow(
+                  variableId = tableColumnVariableRow1.id!!,
+                  variableTypeId = VariableType.Select,
+                  name = "Some other audit type",
+                  position = 2))
+
+      assertEquals(emptyList<String>(), importResult.errors, "no errors")
+      assertEquals(
+          expectedTableColumnRows,
+          actualTableColumnRows,
+          "Variable Table Column DB rows are correct")
+      assertEquals(expectedNumberRow, actualNumberRow, "Variable Number row is correct")
+      assertEquals(expectedSelectRow, actualSelectRow, "Variable Select row is correct")
+      assertEquals(
+          expectedSelectOptionRows,
+          actualSelectOptionRows.map { it.copy(id = null) },
+          "Variable Select Option rows are correct")
+    }
+
+    @Test
+    fun `ensures that single and multi line text are saved into the DB as expected`() {
+      val testCsv =
+          header +
+              "\nOrganization Name,1,,Text (single-line),,,,,,,,,,,,,,," +
+              "\nOrganization Name As List,2,,Text (single-line),yes,,,,,,,,,,,,,," +
+              "\nPrior Scenario,3,A brief description of the scenario,Text (multi-line),,,,,,,,,,,,,,," +
+              "\nPrior Scenario As List,4,A brief description of the scenario,Text (multi-line),yes,,,,,,,,,,,,,,"
+
+      val importResult = importer.import(sizedInputStream(testCsv))
+
+      val actualVariableO = getVariableByName("Organization Name")
+      val actualVariableTextO = variableTextsDao.fetchOneByVariableId(actualVariableO.id!!)
+      val actualVariableOAL = getVariableByName("Organization Name As List")
+      val actualVariableTextOAL = variableTextsDao.fetchOneByVariableId(actualVariableOAL.id!!)
+      val actualVariablePS = getVariableByName("Prior Scenario")
+      val actualVariableTextPS = variableTextsDao.fetchOneByVariableId(actualVariablePS.id!!)
+      val actualVariablePSAL = getVariableByName("Prior Scenario As List")
+      val actualVariableTextPSAL = variableTextsDao.fetchOneByVariableId(actualVariablePSAL.id!!)
+
+      assertEquals(emptyList<String>(), importResult.errors, "no errors")
+
+      Assertions.assertFalse(actualVariableO.isList!!, "single input without list is not a list")
```

Comment:
Should use a static import for this and the other assert methods. When I end up with this (usually after I use IntelliJ to duplicate a class and it decides to replace the existing static imports with qualified names) I usually search for `Assertions.` and walk through the file seeing what assertions I need to import.